lua/methods/unit.lua

```diff
@@ -5,7 +5,7 @@
 
 local Unit = Unit
 local Object = Object
-local next = next 
+local GRPC = GRPC

```

Comment:
I think I didn't ask in the previous PR, why are the local aliases necessary?

Reply:
I am completely cargo-culting this but from what I have read it improves performance.